styles.css

```diff
@@ -127,9 +125,27 @@ footer p {
 }
 
 .ratio-result {
-  margin-right: 2rem;
+  margin: 0 2rem;
+  padding: 1rem;
 }
 
+.level-labels {
+  margin: 1.5rem 0 1rem 0;
+  display: flex;
+  justify-content: space-around;
+  font-variation-settings: "wght" 600;
+}
+
+.level {
+  width: 3rem;
+}
+
+.icons {
+  width: 110px;
```

Comment:
I understand why you've added the `width` here. But, don't worry about it, you can remove it. Issue #85 will have a fix for this.

Reply:
Ohh and to think i wanted  asking before making the change.Alright then.

Reply:
changes have been made please have a look.